src/Qir/check-sources-formatted.ps1

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+#Requires -PSEdition Core
+#Requires -Version 7.1
```

Comment:
These two are redundant, such that only either `#Requires -PSEdition Core` or `#Requires -Version 7.1` are needed, but not both. The version requirement is stricter, since any version after 6.0 counts as "Core", so I would suggest just keeping that one.